src/main/java/uk/gov/crowncommercial/dts/scale/cat/utils/TendersAPIModelUtils.java

```diff
@@ -108,6 +125,63 @@ public EventDetail buildEventDetail(final RfxSetting rfxSetting,
     return eventDetail;
   }
 
+  /*
+   * SCAT-4788 - need to call this to populate EventSummary.dashboardStatus and
+   * EventDetail.nonOCDS.dashboardStatus
+   */
+  public DashboardStatus getDashboardStatus(final RfxSetting rfxSetting,
+                                            final ProcurementEvent procurementEvent) {
+
+    var tenderStatus = procurementEvent.getTenderStatus();
+    if (Objects.nonNull(tenderStatus)) {
+      if (Objects.equals(CLOSED_STATUS, tenderStatus) || Objects.equals(COMPLETE_STATUS, tenderStatus)) {
+        if (Objects.equals(COMPLETE_STATUS, tenderStatus)) {
+          return DashboardStatus.COMPLETE;
+        } else if (Objects.equals(CLOSED_STATUS, tenderStatus)) {
+          return DashboardStatus.CLOSED;
+        }
```

Comment:
I think we could remove the outer `if()` here, seems redundant?

Reply:
agree. Removed